### ISSUE TITLE
npm run build fails if editUrl is not set

### ISSUE THREAD

**Issue by otbe** [boyney123/eventcatalog#93]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/boyney123/eventcatalog/blob/master/CONTRIBUTING.md#reporting-new-issues).

### Description

I think https://github.com/boyney123/eventcatalog/pull/75 introduced an issue with building a prod build.

If I run ` npm run build` it will fail with
```
Failed to compile.

./pages/events/[name].tsx:5:10
Type error: Module '"../../eventcatalog.config"' has no exported member 'editUrl'.

  3 | 
  4 | import { Event } from '@eventcatalog/types';
> 5 | import { editUrl } from '../../eventcatalog.config';
```

### Steps to reproduce

https://github.com/otbe/eventcatalog-playground

### Expected behavior

Should work without editUrl set

### Actual behavior

Fails creating a production build

### Your environment

- EventCatalog version used: 0.0.9 and 0.0.10

**Comment by boyney123** [comment]
Ahh.... thanks @otbe 👍, gonna have to add tests around the react app soon.... one step at time!

**Comment by otbe** [comment]
Yes sure, no pressure. I just want to drop things here as I notice them :)

**Comment by boyney123** [comment]
Should be fixed in `@eventcatalog/core@0.0.11`, will check this out and make sure I can build 😅👍

**Comment by boyney123** [comment]
Cool seems fixed, thanks @otbe (again!)
